File: 3830-find-closest-person/find-closest-person.py
class Solution:
    def findClosest(self, x: int, y: int, z: int) -> int:
        xd = abs(x - z)
        yd = abs(y - z)

        if xd < yd:
            return 1
        elif xd > yd:
            return 2
        else:
            return 0

File: 3830-find-closest-person/find-closest-person.js
/**
 * @param {number} x
 * @param {number} y
 * @param {number} z
 * @return {number}
 */

var findClosest = function (x, y, z) {
    if (Math.abs(z - x) < Math.abs(z - y)) {
        return 1;
    } else if (Math.abs(z - y) < Math.abs(z - x)) {
        return 2;
    } else {
        return 0;
    }
};